setup.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf -8*-
+from setuptools import setup
+
+setup(
+    name="mailroom",
+    description="This package allows the user to update an existing list of"
+                "donors, to generate a Thank-You Letter,"
+                "and to create a report",
```

Comment:
the format of this string will be a bit odd.  There is no space between 'of' in the first line, and 'donors' in the second.  You have to leave one there.